src/common/modules/data/DefaultSettings.js

```diff
@@ -11,12 +11,22 @@
  * @const
  * @type {Object}
  */
+
+// checks for OS dark theme and returns the appropriate background color
+ const setQrBackgroundColor = () => {
+     if (window.matchMedia('(prefers-color-scheme: dark)').matches) {
+        return "#d7d7db"; // Photon Grey 30
+     } else {
+        return "ffffff";
```

Comment:
Embarrassing. Looks like tests worked because white is the default, I tried it in a ternary with `null` and the result was the same. Fixing for readability